src/pages/_app.js

```diff
@@ -135,6 +136,28 @@ Layout.propTypes = {
   children: PropTypes.node,
 };
 
+const Head = () => {
+  const { t } = useTranslation();
+  return (
+    <NextHead>
+      <meta
+        key="viewport"
+        name="viewport"
+        content="initial-scale=1.0, width=device-width"
+      />
+      <link
+        key="icon"
+        rel="icon"
+        type="image/png"
+        sizes="32x32"
+        href="/favicon.png"
+      />
+      <title key="title">UitDatabank</title>
```

Comment:
Should we add a follow-up ticket / PR to add more info to this title, e.g. the title of the current page the user is on? (Might also need changes in AngularJS to push the title of the page to the parent window)

Reply:
We can Improve this in the future, but I asked @corneelwille how important SEO was and we came to the conclusion to not spend too much time on it right now.

Reply:
No of course but it's also just a nice UX improvement for users using multiple tabs (most probably?)